validator-server/src/main/kotlin/science/numcompass/conformance/fhir/chain/TerminologyService.kt

```diff
@@ -11,6 +11,19 @@ import org.hl7.fhir.instance.model.api.IBaseResource
 import org.hl7.fhir.r4.model.ValueSet
 import science.numcompass.conformance.fhir.chain.ValidationChain.IValidator
 
+/**
+ * Try to check if a system-code pair represents a post-coordinated (composite) SNOMED code.
+ *
+ * The current check is very simple and may potentially flag too many codes.
+ */
+internal fun isPostCoordinateSnomed(theCodeSystem: String?, theCode: String?): Boolean {
+    // Check if this is declared as a SNOMED code
+    if (theCode == null || theCodeSystem?.startsWith("http://snomed.info/sct") != true) return false
```

Comment:
```suggestion
    if (theCode == null || theCodeSystem?.startsWith("http://snomed.info/sct") == false) return false
```

Reply:
Thanks for the suggestion. However, this will not work as is since the expression should also evaluate to `true` if the left hand side is `null`. One could use a ``==  false` condition if combined with either a `theCodeSystem == null` check or by using the Elvis operator (`?:`) to replace a `null` with "0". But my feeling is that this would not be more legible - what do you say?

Reply:
Sorry, I was just replacing `!= true` with `== false` isn't that the same ?